apps/schematic/api/schematic_api/controllers/storage_controller.py

```diff
@@ -13,6 +13,23 @@
 from schematic_api.controllers import storage_controller_impl
 
 
+def get_manifest_json(asset_type, manifest_id):  # noqa: E501
```

Comment:
sorry I don't understand how everything in openapi works. is this a whole endpoint to return the manifest as a JSON? Or just the result of selection in the endpoint that could also return the manifest in other outputs?